besca/pl/_nomenclature_network.py

```diff
@@ -0,0 +1,33 @@
+from matplotlib import pyplot as plt
+import pandas as pd
+_nomenclature_network.import importlib
+
+networkx_import = importlib.util.find_spec('networkx')
+pydot_import = importlib.util.find_spec('pydot')
+
+if (networkx_import or pydot_import) is None:
+    raise ImportError(
+        "_nomenclature_network.py requires networkx and pydot. Install with pip install pydot and pip install networkx")
+else:
+    import networkx as nx
```

Comment:
Is Import pydot missing ?

Reply:
`pydot` is imported with `networkx` but is not listed as a dependency of `networkx`. Thus it is not automatically installed with `networkx`.